Cognigy/CognigyWebChat
# Add functionality to display messages sent by client.sendMessage()

We now have a third parameter to the sendMessage function, which is 'displayMessage'. If set to true, then the text will be shown in the Webchat. You can read documentation about that here https://docs.cognigy.com/v3.1.9/docs/integrate-the-webchat-on-your-own-website#section-using-the-cognigyai-client-to-send-messages.

***How to test***
Replace the contents of src/index.html with the following code:

```
<body>
	<script>

		// Replace the configUrl with the configUrl pointing to your Webchat Endpoint
		var configUrl = "https://endpoint-demo.cognigy.ai/e2425d2a9c49843d25b8e346a6331a265b52b2535b25fe3d8550779f8b7bc348";

		var client;

		Cognigy.init(configUrl)
			.then(c => client = c);
	</script>

	<button onclick="client.sendMessage('Hi', {}, true)">Click me!</button>
</body>
```

Afterwards, run ``npm start`` to start the Webchat. Clicking the button should display "Hi" in the Webchat
